public/css/tailwind.css

```diff
@@ -11,7 +11,7 @@
   --color-sii-primary: #644FC1;
   --color-sii-light-accent: #EDE9FE;
   --color-sii-dark-accent: #977FC3;
-  --color-sii-light-hero-background: #fbfbfb;
+  --color-sii-light-hero-background: lime;
```

Comment:
There should be an additional background on the hero image in light theme that matches the same with a background color on the dark theme. I've set this to `lime` for you to see what I mean, since I'm not sure what color it should be.

Reply:
This should remain `#fbfbfb`. there isn't a separate color defined.